suites/portal/homepageTest.js

```diff
@@ -6,3 +6,17 @@ Scenario('login @portal', (home) => {
   home.ask.seeDetails();
   home.complete.logout();
 });
+
+/*
+Data Setup:
+    1. User_1 - Synapse account having access to a project in a brain commons (Commons_1)
```

Comment:
Similarly here, can you add the user details inside the Scenarios as a part of steps?

Reply:
@atharvar28 will do